src/assets/ElmAsset.js

```diff
@@ -0,0 +1,102 @@
+const process = require('process');
+const Asset = require('../Asset');
+const localRequire = require('../utils/localRequire');
+const {minify} = require('terser');
+
+class ElmAsset extends Asset {
+  constructor(name, options) {
+    super(name, options);
+    this.type = 'js';
+    this.hmrPageReload = true;
+  }
+
+  async collectDependencies() {
+    const {findAllDependencies} = await localRequire(
+      'find-elm-dependencies',
```

Comment:
I think we can avoid installing this package directly if we use `elm.findAllDependencies` from the `node-elm-compiler` package, which already depends on `find-elm-dependencies`. This way only one package will end up getting installed in a user's app.